app_config.py

```diff
@@ -1,21 +1,16 @@
 import os
 
-CLIENT_ID = "Enter_the_Application_Id_here" # Application (client) ID of app registration
+# Application (client) ID of app registration
+CLIENT_ID = os.getenv("CLIENT_ID")
```

Comment:
Lots of changes here due to the .env file.

Reply:
Good move. But then please also change the README in this repo accordingly.

And I do not know how many docs currently in docs.microsoft.com and/or learn.microsoft.com would also need to change.

Reply:
I was merely saying that we would need to at least change ["Step 3: Configure the sample to use your Azure AD tenant" of our README here](https://github.com/Azure-Samples/ms-identity-python-webapp#step-3--configure-the-sample-to-use-your-azure-ad-tenant). Optionally, feel free to also simplify its ["Step 2: Register the sample application with your Azure Active Directory tenant"](https://github.com/Azure-Samples/ms-identity-python-webapp#step-2--register-the-sample-application-with-your-azure-active-directory-tenant).

Similarly, the ["Step 3: Configure the sample to use your Azure AD B2C tenant" of our README_B2C](https://github.com/Azure-Samples/ms-identity-python-webapp/blob/master/README_B2C.md#step-3--configure-the-sample-to-use-your-azure-ad-b2c-tenant) also needs adjustment, but you do not need to change its "how to register your application" content, because that content has been covered by its [Prerequisite](https://github.com/Azure-Samples/ms-identity-python-webapp/blob/master/README_B2C.md#prerequisites) pointers.

Reply:
I've updated the README to remove the parts that were redundant with the Learn.com article (and also merged in some parts from the README to the Learn.com PR that seemed helpful). I chatted with the content team and they confirmed it makes sense to remove redundant content, since it's hard to maintain content in two places.
I still need to go through the B2C flow and update that README.